packages/generator-react-server/generators/app/templates/routes.js

```diff
@@ -1,10 +1,11 @@
+const path = require('path');
 
 module.exports = {
 	routes: {
 		HelloWorld: {
 			path: ['/'],
 			method: 'get',
-			page: './pages/hello-world',
+			page: path.join('.', 'pages', 'hello-world'),
```

Comment:
:squint:

I'm so confused by windows.

I thought forward slashes were actually okay now, but you might _receive_ backslashes from getters?